src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleList } from '../PeopleList/PeopleList';
+
+function getFixedPeople(people: Person[]) {
+  const peopleCopy: Person[] = JSON.parse(JSON.stringify(people));
+
+  peopleCopy.forEach(person => {
+    // eslint-disable-next-line no-param-reassign
+    person.mother = people
+      .find(mother => mother.name === person.motherName);
+    // eslint-disable-next-line no-param-reassign
+    person.father = people
+      .find(father => father.name === person.fatherName);
```

Comment:
Try to refactor this solution to avoid disabling linter

P.S. you can use `map` here and return copy `{..., mother: '', father: ''}`